README.md

```diff
@@ -95,6 +95,7 @@ Python course would cover.
 2. **TIC-TAC-TOE**
 3. **Rock-Paper-Scissors (3-way and 7-way)**
 4. **A Connect4 clone**
+5. **Snake Game**
```

Comment:
We didn't cover a snake game did we? What part are you referring to?

Reply:
Actually I am referring to adding **Snake Game** in this Course.So it'll cover basic and intermediate Concept.